### DISCUSSION TITLE
Nemesis crashes (osx?)

### DISCUSSION THREAD

**Discussion by veeshy** [idaholab/moose#20844] [Q&A Meshing] [answered]
## Bug Description
Using nemesis output causes some error where moose crashes. If the execute_on is only once at the end, this doesn't seem to cause the output to be wrong, but you still get an error message that doesn't seem to kill moose so tests pass (I'm speculating..). If you try to execute any nemesis test on initial it won't run.

## Steps to Reproduce
in the moose tests folder:
```
./moose_test-opt -I ./tests/outputs/nemesis/nemesis_elemental.i Outputs/execute_on=initial
```
gives me

```
zsh: illegal hardware instruction  ./moose_test-opt -i  Outputs/execute_on=initial
```

Not setting the initial gives me the same as above but moose does run and the test associated does pass. 

Trying to run with `mpiexec -n 4` causes an actual bad termination. 

My moose:
```
Framework Information:
MOOSE Version:           git commit 668cd38e2e on 2020-07-07
LibMesh Version:         6e07d0a45166892bdbeef78440bad7aa46e2a5b7
PETSc Version:           3.12.5
SLEPc Version:           3.12.1
```

I'm on OS X `10.15.5 (19F101)` using the standard conda moose. I haven't tried to replicate in ubuntu.

## Impact
I can't run distributed mesh with a parallel output.

**Comment by GiudGiud** [comment, marked as answer]
@veeshy is this fixed? or did you update the mac and now it s gone?

**Reply by veeshy** [reply]
Do not have the specific versions listed here, happy to check next week though on a newer mac.

**Reply by veeshy** [reply]
this is fixed (sorry for the delay)

osx 12.3, latest moose

**Reply by GiudGiud** [reply]
great thank you for checking!
